<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="startYearSpinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;시작연도를 선택합니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="endYearSpinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;종료연도를 선택합니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="startYearLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시작연도</string>
    </property>
   </widget>
   <widget class="QLabel" name="endYearLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>종료연도</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="repeatNumSpinBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="repeatNum">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>몇 일마다 반복할까</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dirSelectButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>경로선택</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="dirText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dirLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>생성 경로</string>
    </property>
   </widget>
   <widget class="QPushButton" name="letsGetit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>390</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>레스기릿</string>
    </property>
   </widget>
   <widget class="QLabel" name="descLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>일정 이름</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="description">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
